cudamapper/src/overlapper.cpp

```diff
@@ -171,6 +196,24 @@ void Overlapper::post_process_overlaps(std::vector<Overlap>& overlaps)
         fused_overlap.num_residues_                  = num_residues;
         overlaps.push_back(fused_overlap);
     }
+
+    if (drop_fused_overlaps)
```

Comment:
Without a zip iterator I don't think I can use `std::remove_if` as this is (from the std::remove_if docs: " accepts an element in the range as argument"). We could put the overlaps / references in a set but this might use more memory than the boolean mask (which we could make an explicit bit set if we absolutely needed, though we'd need a non-standard implementation).

Reply:
Yes, let's leave it as it is then.